method-missing.rb

```
@@ -0,0 +1,11 @@
+# Using method_missing to handle dynamic method calls
+class DynamicHandler
+    def method_missing(name, *args)
+        if name.to_s.start_with?('handle_')
+            # Custom handling logic
+            puts "Handling #{name}"
+        else
+            super
+        end
+    end
```

RuboCop (1.69.1)

[convention] When using `method_missing`, define `respond_to_missing?`.

(Style/MissingRespondToMissing)